powerbi-docs/connect-data/desktop-connect-oracle-database.md

```diff
@@ -34,6 +34,8 @@ To determine which version of Power BI Desktop is installed, select **File** > *
 - For the 64-bit version of Power BI Desktop, [download and install the 64-bit Oracle client](https://www.oracle.com/database/technologies/odac-downloads.html).
 
 > [!NOTE]
+> Ensure you choose a version of Oracle Data Access Client (ODAC) which is compatible with your Oracle Server. For instance, ODAC 12.x does not support Oracle Server version 9.
```

Comment:
Sorry this statement I gave to you ealier is not accurate.  It should be:

ODAC 12.x does not always work with Oracle Server version 9.

Reply:
Changed to "...does not always work..."